docs2/site/docs/migrations/migration5.md

```diff
@@ -196,8 +202,44 @@ If you use provided extension methods upon `IGraphQLBuilder` then your code does
 
 ### 13. Changes caused by GraphQL-Parser v8
 
+- The `GraphQL.Language.AST` namespace and all classes from it have been removed in favor of ones
+  from `GraphQLParser.AST` namespace in GraphQL-Parser project. Examples of changed usages:
+  - `GraphQL.Language.AST.Document` -> `GraphQLParser.AST.GraphQLDocument`
+  - `GraphQL.Language.AST.IValue` -> `GraphQLParser.AST.GraphQLValue`
+  - `GraphQL.Language.AST.Field` -> `GraphQLParser.AST.GraphQLField`
+  - `GraphQL.Language.AST.SelectionSet` -> `GraphQLParser.AST.GraphQLSelectionSet`
+  - `GraphQL.Language.AST.IHaveDirectives` -> `GraphQLParser.AST.IHasDirectivesNode`
+  - `GraphQL.Language.AST.IType` -> `GraphQLParser.AST.GraphQLType`
+- Some APIs become to work with `GraphQLParser.ROM` struct instead of `string`:
```

Comment:
Needs rewording - for example:

- Some APIs utilize `GraphQLParser.ROM` struct instead of `string`:

Reply:
ok